package.json

```diff
@@ -42,7 +42,7 @@
     "size:why": "size-limit --why packages/material-ui/build/index.js",
     "size:overhead:why": "size-limit --why ./test/size/overhead.js",
     "test": "yarn lint && yarn typescript && yarn test:coverage",
-    "test:unit": "cross-env NODE_ENV=test mocha 'packages/**/*.test.js' 'docs/**/*.test.js'",
+    "test:unit": "cross-env NODE_ENV=test mocha 'packages/**/*.test.js' 'docs/**/*.test.js' --exclude '**/node_modules/**'",
```

Comment:
https://github.com/cssinjs/jss/issues/979

Reply:
The linked issue is about publishing tests which is another issue as far as I can tell.

This will only exclude a tiny portion of `node_modules` since most of them should be hoisted to the workspace root instead of the individual workspaces. I would defer this to another PR and to it "right" i.e. apply it to every workspace and not just the root.

Reply:
I'm fixing the other packages. It's a small change. I don't want to create noise in the history.